src/actions/mailing/sendProfileStatusEmail.tsx

```diff
@@ -0,0 +1,36 @@
+'use server'
+
+import { mailersendClient, MailTemplateId } from '@/lib/mailersendClient'
+import { PublishingState } from '@prisma/client'
+import { Recipient } from 'mailersend'
+
+export const sendProfileStatusEmail = async (
+  email: string,
+  status: keyof typeof PublishingState,
+) => {
+  function setTemplateByStatus(status: keyof typeof PublishingState) {
```

Comment:
**issue:** This should not be a single method that manages different statuses, because logic can differ a lot between them. We should make explicit actions, like: `sendProfileApproval`, and `sendProfileRejection` because `sendProfileRejection` needs to get data about rejection reason that is created in DB, and it should send it to the user so he knows why he was rejected